Makefile

```diff
@@ -169,8 +169,12 @@ push:
 
 .PHONY: sign
 sign:
+	docker pull $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG)
+	docker pull $(DOCKER_IMAGE_NAME):$(MUTABLE_DOCKER_TAG)
```

Comment:
Since buildx builds/pushes the images from a sidecar, this makes the images available locally for signing in the primary container.